packages/webdriverio/src/commands/element/addValue.ts

```diff
@@ -1,9 +1,21 @@
+import { DeprecatedWarning } from '../../utils/DeprecatedWarning'
 import { transformToCharString } from '../../utils'
 
-export type AddValueOptions = {
+export type CommandOptions = {
     translateToUnicode?: boolean
 }
 
+export type Value = string | number
+
+function isValidType(value: unknown) {
+    const isNumberOrString = (input: unknown) => typeof input === 'string' || typeof input === 'number'
+
+    return (
```

Comment:
Let's define this function only once.

```suggestion
export type Value = string | number
const isNumberOrString = (input: unknown) => typeof input === 'string' || typeof input === 'number'

function isValidType(value: unknown) {
    return (
```